v1/tarball/image_test.go

```diff
@@ -15,11 +15,70 @@
 package tarball
 
 import (
+	"archive/tar"
+	"io"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/google/go-containerregistry/name"
 )
 
+func TestFlatten(t *testing.T) {
+	img, err := ImageFromPath("testdata/whiteout.tar", nil)
```

Comment:
Would it be possible to construct this with bazel and reference it as a dependency of the test, instead of checking it in? It would make the contents/functionality of this test more transparent.